src/index.html

```diff
@@ -177,7 +177,7 @@
                  data-chart-color="blue"
                  data-chart-description="This is the chart description."
                  data-chart-metadata="High"
-                 data-chart-source="consumer-credit-trends/volume_data_Income_Level_MTG.csv">
+                 data-chart-source="consumer-credit-trends/mortgage/volume_data_Income_Level_MTG.csv">
```

Comment:
Same as stated earlier. Please make the market folders plural, to match up with the naming on the website: `mortgages`